src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -222,6 +252,16 @@
   @Fluent
   MqttEndpoint publishAcknowledgeHandler(Handler<Integer> handler);
 
+  /**
+   * Set the puback handler on the MQTT endpoint. This handler is called when a PUBACK
+   * message is received by the remote MQTT client
+   *
+   * @param handler the handler
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  MqttEndpoint publishAcknowledgeMessageHandler(Handler<MqttPubAckMessage> handler);
```

Comment:
I'm wondering if we should not bame it `PubAckMesssageHandler` for consistency

Reply:
There's `MqttEndpoint publishAcknowledgeHandler(Handler<Integer> handler);` method in this interface. We'd probably need to rename it too in this case.

Reply:
right this is a good point.